### ISSUE TITLE
[Bug]: the perf decreased for sysbench oltp update,insert,delete test on eks on daily run 

### ISSUE THREAD

**Issue by aressu1985** [matrixorigin/matrixone#9673]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):nightly-6a65b65
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

for daily run on 5.24 and 5.25 on eks, the perf of sysbench write cases decrease very seriously.

<img width="1201" alt="image" src="https://github.com/matrixorigin/matrixone/assets/47846308/f696607d-12ef-4c0d-a423-69a72470bd82">

but I can not repro, and no proflie can be provided. If repored, I will provide info as soon as I can

### Steps to Reproduce

```Markdown
It it very hard to repro
```

**Comment by aressu1985** [comment]
[insert-cn-0-10.0.107.2.zip](https://github.com/matrixorigin/matrixone/files/11570817/insert-cn-0-10.0.107.2.zip)
[insert-cn-1-10.0.123.134.zip](https://github.com/matrixorigin/matrixone/files/11570818/insert-cn-1-10.0.123.134.zip)
[update-cn0-10.0.107.2.zip](https://github.com/matrixorigin/matrixone/files/11570820/update-cn0-10.0.107.2.zip)
[update-cn1-10.0.123.134.zip](https://github.com/matrixorigin/matrixone/files/11570821/update-cn1-10.0.123.134.zip)

**Comment by cnutshell** [comment]
> [insert-cn-0-10.0.107.2.zip](https://github.com/matrixorigin/matrixone/files/11570817/insert-cn-0-10.0.107.2.zip) [insert-cn-1-10.0.123.134.zip](https://github.com/matrixorigin/matrixone/files/11570818/insert-cn-1-10.0.123.134.zip) [update-cn0-10.0.107.2.zip](https://github.com/matrixorigin/matrixone/files/11570820/update-cn0-10.0.107.2.zip) [update-cn1-10.0.123.134.zip](https://github.com/matrixorigin/matrixone/files/11570821/update-cn1-10.0.123.134.zip)

从 update-cn0 堆栈来看，cn0 节点中 IO wait 状态的 goroutine 数量较多，不确定 morpc.(*remoteBackend) readLoop 与 tcp 连接是否为 1:1 的对应关系，不过这意味着大量的 tcp 连接，这可能会造成性能降级。
根据目前已有的 profile，无法进一步判断性能降级的原因，也许我们需要考虑提供服务运行指标，通过对比降级前后服务运行状态来协助分析。
<img width="1102" alt="image" src="https://github.com/matrixorigin/matrixone/assets/20291742/c88a9cba-c8fd-4118-9823-cedafbe8fec5">
<img width="643" alt="image" src="https://github.com/matrixorigin/matrixone/assets/20291742/f1810b3b-0d5d-43d4-9e75-b0b5094a6df2">

**Comment by cnutshell** [comment]
可以通过 dn 日志或者 dn 相关 profile 进一步分析性能降级的原因。

**Comment by aptend** [comment]
this is not a case that can be repro steadily

**Comment by aptend** [comment]
working on s-1

**Comment by aptend** [comment]
no repro, consider closing

**Comment by aressu1985** [comment]
fixed:nightly-d4d74e7
